test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -2260,7 +2281,7 @@ public class Microsoft.OData.Edm.EdmComplexTypeReference : Microsoft.OData.Edm.E
 	public EdmComplexTypeReference (Microsoft.OData.Edm.IEdmComplexType complexType, bool isNullable)
 }
 
-public class Microsoft.OData.Edm.EdmCoreModel : Microsoft.OData.Edm.EdmElement, IEdmElement, IEdmModel, IEdmValidCoreModelElement {

```

Comment:
This looks like a breaking change as IEdmValidCoreModelElement  and IEdmCoreModelElement are not related. Did you intend to derive IEdmValidCoreModelElement from IEdmCoreModelElement as well?

Reply:
Thanks for catch this. 

However, `IEdmValidCoreModelElement` is an internal interface.

I renamed `IEdmValidCoreModelElement` as `IEdmCoreModelElement`.  Is Renaming an interface name a breaking change?  (I don't think so. :)  )

publicApi test shows this as a breaking change. However, I think publicApi.test should skip the internal interface. What do you think?

Reply:
Thanks, I marked it as #Won't Fix.